docs/user/cassandra/install.rst

```diff
@@ -40,12 +46,12 @@ installation. To set the variable add the following line to your ``.profile`` or
 
     export CASSANDRA_HOME=/path/to/cassandra
 
-Finally, make sure you know a contact point for your Cassandra instance.
-If you are just trying things locally, and using the default Cassandra settings,
+Finally, make sure you know a contact point for your Cassandra / ScyllaDB instance.
+If you are just trying things locally, and using the default Cassandra / ScyllaDB settings,
 the contact point would be ``127.0.0.1:9042``. You can check and configure the
-port you are using using the ``native_transport_port`` in the Cassandra
+port you are using using the ``native_transport_port`` in the Cassandra / ScyllaDB
 configuration file (located at ``conf/cassandra.yaml`` in your Cassandra
-installation directory).
+installation directory, ``/etc/scylla/scylla.yaml`` for ScyllaDB accordingly).
```

Comment:
Have you tested ScyllaDB with GeoMesa?  

Is there a way that GeoMesa committers can test with ScyllaDB?

Reply:
Hey @jnh5y , 

Yes, I've tested ScyllaDB with GeoMesa. Cassandra quick start example works, all you need to do is to have running ScyllaDB on `127.0.0.1:9042`(same default port) instead of Cassandra.

Regarding tests: original Cassandra module tests are using an embedded Cassandra server. This could be migrated to `testcontainers`, so we would run the same tests both for Cassandra and ScyllaDB. WDYT?

Reply:
switching the cassandra module to use testcontainers would be nice - we just haven't gotten around to doing so.